front/app/components/Form/index.tsx

```diff
@@ -146,6 +193,48 @@ const Form = memo(
       [getApiErrorMessage]
     );
 
+    // Modify required to show the current locale as required.
```

Comment:
Quirk number 4 : The Multilocs
To handle multilocs we had the two options of adding one control for each multiloc thing : InputMultiloc, WYSIWYGMultiloc, or have the top-level multiloc object be a custom layout that shows the appropriate field and render the controls inside normally. I went for the second option.
Both options limited somehow the validation power, and with this solution, it means that the errors on the layout level are not available (IE this field is required, or this field should have at least one property). So this is a hacky thing to make the current locale required, but we will have to find something better would we want to make all locales required like in the admin side or simply is we would want to have a cleaner form component.

Reply:
Could you add this explanation as a comment?